_helpers/generate-env-variable-manifest.js

```diff
@@ -19,9 +19,18 @@ async function writeResult(templates) {
 }
 
 async function getVariablesForTemplate(templateId) {
-  const envFilePath = path.resolve(__dirname, '..', templateId, '.env');
-  const result = await parser.parseFile(envFilePath);
-  return result.variables;
+  try {
```

Comment:
can you update https://github.com/twilio-labs/function-templates/tree/main/_helpers/meta-template and https://github.com/twilio-labs/function-templates/tree/main/blank to have `.env.example` instead of `.env` and possibly also adding a .gitignore file for `.env`

Reply:
I added `.env.example` to the `blank` template and `.env` to `.gitignore`. I think `meta-template` had it already as part of an earlier commit, but if I left out part of the update let me know!